docs/src/literate/stokes-flow.jl

```diff
@@ -461,7 +462,7 @@ function main()
     vtk_grid("stokes-flow", grid) do vtk
         vtk_point_data(vtk, dh, u)
     end
-    Sys.isapple() || @test norm(u) ≈ 0.32254330524111213 #src
+    Sys.isapple() || @test norm(u) ≈ 0.322804206709586 #src
```

Comment:
Good catch. I wonder if this fixes the example on macOS:
```suggestion
    @test norm(u) ≈ 0.322804206709586 #src
```

Reply:
Btw, do you know under which circumstances the mean value constraint is valid? It seems like we require no slip in a Stokes problem to make it work, but I cannot find material on the theory.